flow/aggregates.py

```diff
@@ -128,9 +134,11 @@ def aggregator_function(jobs):
             args = [iter(jobs)] * num
             return zip_longest(*args)
 
-        setattr(aggregator_function, '_num', num)
+        # Special attribute for default aggregates in flow (Groups of 1)
+        if sort_by is None and select is None and not reverse_order:
+            setattr(aggregator_function, '_groupby_is_aggregate', num != 1)
 
-        return cls(aggregator_function, sort_by, reverse_order, select)
+        return cls(aggregator_function, sort_by, reverse_order, select, f'predefined-groupsof{num}')
```

Comment:
What I meant was that we can create the object and if it is the default aggregation, change the `_is_aggregate` flag after instantiation.

Reply:
I pushed a code, please have a look. I hope that's what you meant.